definitions/npm/@apollo/react-common_v3.x.x/flow_v0.104.x-/react-common_v3.x.x.js

```diff
@@ -669,13 +669,14 @@ declare module '@apollo/react-common' {
     ...,
   };
 
-  declare class ApolloError extends Error {
+  declare class $ApolloError extends Error {
```

Comment:
Let's update your fork's `master` branch to be inline with the upstream `master`. That should remove these merge artifacts.